src/app/rundown/components/editorial-line/editorial-line.component.scss

```diff
@@ -0,0 +1,7 @@
+:host {
+  top: var(--toolbar-height);
+  bottom: 0;
+  border-left: 2px dashed grey;
+  margin-left: -1px;
+  position: absolute;
```

Comment:
Minor thing. It would be nice to the structural attributes as the first entries – so in this case position at the top. The (incomplete) order I personally use  is:
- Structural (position, display, ...)
- Placement and sizing
- Colors
- Text formatting

This way, the higher up the list you are, the higher impact it has on the layout.